Source/Examples/Xamarin.Forms/SimpleDemo2/SimpleDemo2/MainPage.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows.Input;
+using OxyPlot;
+using OxyPlot.Axes;
+using OxyPlot.Series;
+using OxyPlot.Xamarin.Forms;
+using Xamarin.Forms;
+
+namespace SimpleDemo2
+{
+	public class MainPage : ContentPage
+	{
+		private Command<IEnumerable<HitTestResult>> _tapCommand = null;
+		public ICommand TapCommand {
+			get {
+				_tapCommand = _tapCommand ?? new Command<IEnumerable<HitTestResult>>(DoTapCommand);
+				return (_tapCommand);
+			}
+		}
+
+		public MainPage()
+		{
+			var plotModel = new PlotModel {
+				Title = "OxyPlot in Xamarin.Forms",
+				Subtitle = string.Format("OS: {0}, Idiom: {1}", Device.OS, Device.Idiom),
+				Background = OxyColors.LightYellow,
+				PlotAreaBackground = OxyColors.LightGray
+			};
+			var categoryAxis = new CategoryAxis { Position = AxisPosition.Bottom };
+			var valueAxis = new LinearAxis { Position = AxisPosition.Left, MinimumPadding = 0 };
+			plotModel.Axes.Add(categoryAxis);
+			plotModel.Axes.Add(valueAxis);
+			var series = new ColumnSeries();
+			series.Items.Add(new ColumnItem { Value = 3 });
+			series.Items.Add(new ColumnItem { Value = 14 });
+			series.Items.Add(new ColumnItem { Value = 11 });
+			series.Items.Add(new ColumnItem { Value = 12 });
+			series.Items.Add(new ColumnItem { Value = 7 });
+			plotModel.Series.Add(series);
+
+			Padding = new Thickness(0, 20, 0, 0);
+			PlotView plotV = new PlotView {
+				Model = plotModel,
+				Controller = new PlotController(),
+				VerticalOptions = LayoutOptions.Fill,
+				HorizontalOptions = LayoutOptions.Fill,
+			};
+			//Rebind touch to be TapTouchManipulator that will call an ICommand on a tap gesture
```

Comment:
I think it would be better to add this example to the "IssueDemos" project. Let us try to keep the SimpleDemos as simple as possible. And I think SimpleDemo (with portable project) and SimpleDemo2 (with shared project) should show the same plots.